### PR TITLE
ApplicationGroup Tests

### PR DESCRIPTION
**Pull request by schaudhari6254888**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by fengzhou-msft** [comment]
@schaudhari6254888 please run `eng\scripts\Export-API.ps1 eventhub` to update the API file and also re-record all tests since this PR bumped the API version.

**Comment by schaudhari6254888** [comment]
@fengzhou-msft I have tried syncing the main branch and running scripts and pushed. But CI's still fail.

**Comment by damodaravadhani** [comment]
@schaudhari6254888, Can you rename ApplicationGroupData, ApplicationGroupResource and ApplicationGroupCollection class EventHubsApplicationGroupData, EventHubsApplicationGroupResource and EventHubsApplicationGroupCollection.

File: sdk/eventhub/Azure.ResourceManager.EventHubs/api/Azure.ResourceManager.EventHubs.netstandard2.0.cs [hunk 1]
```diff
@@ -875,4 +929,189 @@ public partial class EventHubsSku
...
+        public Azure.ResourceManager.EventHubs.Models.ProvisioningIssueProperties Properties { get { throw null; } }
+    }
+    public partial class ProvisioningIssueProperties
+    {
+        internal ProvisioningIssueProperties() { }
+        public string Description { get { throw null; } }
+        public string IssueType { get { throw null; } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct PublicNetworkAccess : System.IEquatable<Azure.ResourceManager.EventHubs.Models.PublicNetworkAccess>
```
Comment (fengzhou-msft) [inline sdk/eventhub/Azure.ResourceManager.EventHubs/api/Azure.ResourceManager.EventHubs.netstandard2.0.cs:1053]: Consider adding `EventHubs` prefix to model name if the model name is quite common and can appear in other RPs as well like `PublicNetworkAccess`, `ThrottlingPolicy`, `TlsVersion`.

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**